pages/playground.tsx

```diff
@@ -8,7 +8,7 @@ import { PageScaffoldData } from '../components/DefaultLayout';
 import { EChartsBaseChart } from '../components/EChartsBaseChart';
 import { toBasicAxisData } from '../components/EChartsBaseChart/utils';
 import { Legend, LegendItem } from '../components/Legend';
-import { PageSection } from '../components/PageSection';
+import { PageSection, PageSectionHeading, PageSectionSubHeading } from '../components/PageSection';
 import { SpotlightMenuItem } from '../components/SpotlightMenu';
 import { SidebarContent, SidebarHeading, SpotlightSidebar } from '../components/SpotlightSidebar';
 import { SpotlightTab } from '../components/SpotlightTab';
```

Comment:
Don't put this feature in the playground. I need it plugged into the actual place it's meant to appear, even if it's not taking in any data. Review existing sections to see what shape your props are going to take.

Also, like I said before, this is for the revenue and expenditure sections ... I'm already on the indicator comparison section.